shared/src/main/scala/mlscript/codegen/Typegen.scala

```diff
@@ -0,0 +1,74 @@
+package mlscript
+
+/**
+  * Convert types to typescript declarations
+  */
+sealed trait TypeSourceCode { self: Type => 
+
+    def toSourceCode: SourceCode = this match {
```

Comment:
```suggestion
  def toSourceCode: SourceCode = this match {
```
The standard indentation width in Scala is 2 spaces.

Reply:
Fixed.